spring-cloud-aws-sqs/src/main/java/io/awspring/cloud/sqs/annotation/SqsListenerAcknowledgementMode.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2013-2023 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.awspring.cloud.sqs.annotation;
+
+import io.awspring.cloud.sqs.listener.acknowledgement.handler.AlwaysAcknowledgementHandler;
+import io.awspring.cloud.sqs.listener.acknowledgement.handler.NeverAcknowledgementHandler;
+import io.awspring.cloud.sqs.listener.acknowledgement.handler.OnSuccessAcknowledgementHandler;
+
+/**
+ * Configures the acknowledgement behavior for the container.
+ *
+ * @author Joao Calassio
+ * @since 3.1
+ * @see OnSuccessAcknowledgementHandler
+ * @see AlwaysAcknowledgementHandler
+ * @see NeverAcknowledgementHandler
+ * @see io.awspring.cloud.sqs.listener.ContainerOptions
+ * @see SqsListener
+ */
+public enum SqsListenerAcknowledgementMode {
+
+	/**
+	 * Use acknowledge mode defined by the container.
+	 */
+	INHERIT,
```

Comment:
Not sure about `INHERIT`, how about we call it `DEFAULT`?

Another thing is - enums don't really play nice with annotations because we can't use SPeL for them.

I'm thinking if it would make sense to have the `AcknowledgementMode` as a `String` and provide users with a Constant class.

WDYT?

Reply:
yeah, "default" is better. I was thinking something like "inherit from the container (factory)".

I agree with using the const strings instead of enums -- I didn't really think about SpEL, thanks!!

I'll change it